localstack/services/kinesis/kinesis_listener.py

```diff
@@ -59,6 +60,29 @@ def return_response(self, method, path, data, headers, response):
                 event_records.append(event_record)
             stream_name = data['StreamName']
             lambda_api.process_kinesis_records(event_records, stream_name)
+        elif action == ACTION_UPDATE_SHARD_COUNT:
+            # Currently kinesalite, which backs the Kinesis implementation for localstack, does
+            # not support UpdateShardCount:
+            # https://github.com/mhart/kinesalite/issues/61
+            #
+            # [Terraform](https://www.terraform.io) makes the call to UpdateShardCount when it
+            # applies Kinesis resources. A Terraform run fails when this is not present.
+            #
+            # The code that follows just returns a successful response, bypassing the 400
+            # response that kinesalite returns.
+            #
+            response = Response()
+            response.status_code = 200
+            content = {
+                'CurrentShardCount': 1,
+                'StreamName': data['StreamName'],
+                'TargetShardCount': data['TargetShardCount']
+            }
+            response.encoding = 'UTF-8'
+            response.headers = headers
```

Comment:
Not sure this is needed/appropriate here - `headers` contains the _request_ headers, not _response_ headers. I think line 82 should be removed.